src/dymaptic.GeoBlazor.Core/Components/Renderers/Renderer.cs

```diff
@@ -72,6 +73,7 @@ public enum RendererType
 {
 #pragma warning disable CS1591
     Simple,
-    UniqueValue
+    UniqueValue,
+    RasterStretch
```

Comment:
I think this should be removed and not used, since we discovered there is no inheritance. You could replace with just a string entry.